README.md

```diff
@@ -57,6 +56,22 @@ a request is allowed to continue being processed. When the PerimeterX Enforcer d
 ## <a name="upgradingVersions"></a> Upgrading
 See the full [changelog](CHANGELOG.md) for all versions.
 
+To upgrade the PerimeterX Enforcer: 
+
+1. Copy the configuration file [(pxconfig.lua)](lib/px/pxconfig.lua) from the latest source version to your application.
+2. Modify the new configuration file, making sure that it reflects your current/old configuration.
+3. Copy the new file to `/usr/local/lib/lua/px/pxconfig.lua`. (default location). If you wish to change the directory, you must save the file to where the module was previously installed. 
+
+### Custom Block Pages
+
```

Comment:
Should be linked to https://console.perimeterx.com/docs/server_integration_new.html#custom-captcha-section as well imo...

Reply:
it's not supported in this way yet...